src/tests/baseservices/CET/CheckCETPresence.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+static class Program
+{
+    public static int Main()
+    {
+        Console.WriteLine("Checking whether codeflow enforcement technology (CET) is active");
+        // TODO: perform the actual CET check
```

Comment:
The strongest check would be to actually trigger a crash by performing an operation that is prohibited with CET.

We can then have a pair of tests: One that verifies that the operation is succeeding when CET is disabled, and one that verifies that the operation is crashing when CET is enabled.

Reply:
> trigger a crash by performing an operation that is prohibited with CET.

You probably still need to a bit of unmanaged code for that.

Reply:
I think checking the value of SSP is sufficient. The SSP is always nonzero if CET is enabled for the process and zero otherwise.

Reply:
I have already implemented the SSP check. We can easily add more tests if desired once that's shown to work.